docs/docs/use-cases/flux.md

```diff
@@ -0,0 +1,192 @@
+
+This section shows how to add Keptn
+[pre- and post-deployment tasks](../guides/tasks.md)
+to an existing Flux use case that runs
+[pre and post-deployment jobs with Flux](https://fluxcd.io/flux/use-cases/running-jobs/).
+Adding Keptn makes it simpler and more straight-forward
+to run the Flux pre and post-deployment jobs
+and provides added observability out of the box.
+
+## High-level structure of the Git repository
+
+Flux uses a GitOps approach for continuous delivery.
+The Git
+repository structure for our use case looks like the following:
+
+```markdown
+├── apps
+│   └── dev
+│       ├── podtato-head.yaml
+│       └── kustomize.yaml
+└── clusters
+    └── dev
+        ├── flux-system
+        │   ├── gotk-components.yaml
+        │   ├── gotk-sync.yaml
+        │   └── kustomize.yaml
+        ├── podtato-head-source.yaml
+        └── podtato-head-kustomization.yaml
+```
+
+The `apps` directory contains application manifests, that will be deployed.
+The `clusters` directory contains Flux configuration manifests and custom
+resources, that are needed for the delivery.
+The `apps` and `clusters` directories can live in two separate repositories,
+but for simplicity of this excercise, we will keep them in a single one.
+
+## Set up your environment
+
+Before we start, you need to install Flux CLI and Keptn.
+You can find the installation instructions for Keptn [here](../installation/index.md)
+and for Flux [here](https://fluxcd.io/flux/installation/).
+
+After successfully installing Flux CLI and Keptn, you need to
+retrieve your Git repository credentials.
+You can use any available Git providers, but be sure to store your `token`
+for later usage.
+For simplicity, we will use GitHub.
+
+You also need to install Flux in your cluster.
+This requires
+[bootstrapping the Git repository](https://fluxcd.io/flux/installation/bootstrap/)
+to set up all necessary Flux structures in the repository.
+Use the following command to do this:
+
+```shell
+GITHUB_USER=<user-name> GITHUB_TOKEN=<token> \
+flux bootstrap github \
+  --owner=$GITHUB_USER \
+  --repository=podtato-head \
+  --branch=main \
+  --path=./clusters/dev \
+  --personal
+```
+
+The bootstrap command above does the following:
+
+* Creates a Git repository `podtato-head` on your GitHub account.
+* Adds Flux component manifests to the repository and
+creates `./clusters/dev/flux-system/*` structure.
+* Deploys Flux components to your Kubernetes Cluster.
+* Configures Flux components to track the path `./clusters/dev/` in the repository.
+
+## Creating the application
+
+Now it's time to add the Keptn application that defines the Keptn pre- and
+post-deployment checks to the repository.
+
+First, clone the `podtato-head` repository to your local machine.
+
+Add the following manifests that represent the application into the  `podtato-head.yaml` file
+and store it in the `./apps/dev/` directory of your repository:
+
+```yaml
+{% include "./assets/flux/app.yaml" %}
+```
+
```

Comment:
```suggestion

See the Keptn
[CRD reference](../reference/crd-reference/index.md)
documentation for more information about the Keptn resources that are used.
See
[Basic annotations](../guides/integrate.md#basic-annotations)
for a description of the labels and annotations that Keptn uses.
```
How is this for a lightweight way to help the reader find the relevant docs?  You do link to the Flux Kustomize page, etc.  Just a thought -- dismiss if you don't agree.